include/neo_relayboard_v3/RelayBoardV3.h

```diff
@@ -66,6 +66,7 @@ class RelayBoardV3 : public RelayBoardV3Base{
 	bool board_initialized = false;
 	std::map<std::tuple<std::string, std::string, std::string>, std::shared_ptr<const pilot::Incident>> incident_map;
 	std::shared_ptr<const pilot::PowerState> m_power_state;
+	std::vector<std::string> m_system_error;
```

Comment:
This member is not needed. You can use a local variable in `handle(std::shared_ptr<const pilot::SystemState> value)`.

Reply:
https://github.com/neobotix/neo_relayboard_v3/blob/221d93480fd416b146797ebb300082621f060424/src/RelayBoardV3.cpp#L402

But I want to publish the errors here.

Reply:
Err, right, my bad.